pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -472,47 +464,69 @@ func (m *managerImpl) localStorageEviction(pods []*v1.Pod) []*v1.Pod {
 			continue
 		}
 
-		if m.containerOverlayLimitEviction(podStats, pod) {
+		if m.containerScratchLimitEviction(podStats, pod) {
 			evicted = append(evicted, pod)
 		}
 	}
 
 	return evicted
 }
 
+func getPodContainersTotalStorageLimits(pod *v1.Pod) resource.Quantity {
+	containersTotalStorageLimits := resource.Quantity{Format: resource.BinarySI}
```

Comment:
I think you can resue function PodRequestsAndLimits in helpers.go to get the pod-level resources